dev/doc/changes.md

```diff
@@ -1,3 +1,11 @@
+## Changes between Coq 8.13 and Coq 8.14
+
+Gramlib
+
+- A few functions change their interfaces to take benefit of a new abstraction level `LStream` for streams with location function.
+
+  Warning: the type of `Pcoq.Entry.of_parser` changed from `string -> (Plexing.location_function -> te LStream.t -> 'a) -> 'a t` to `string -> (te LStream.t -> 'a) -> 'a t` but OCaml's type-checker generally do not detect it. Existing files compile but then typically raise a stack overflow when parsing. The changes to calls to `Pcoq.Entry.of_parser` have then to be done by "manual" scrutinization.
```

Comment:
*the* OCaml type-checker generally *does not*

Reply:
Actually this comment is outdated, so you can probably remove it altogether.

Reply:
I removed the outdated comment.